e2e/driver/AouElement.ts

```diff
@@ -1,75 +1,115 @@
-import {ClickOptions, ElementHandle, Page} from 'puppeteer';
+import { ElementHandle, Page } from 'puppeteer';
 
 export default class AouElement {
+
   public eHandle: ElementHandle;
 
   constructor(handle: ElementHandle) {
     this.eHandle = handle;
   }
 
-  public async getProperty(property: string): Promise<unknown> {
-    return await (await this.eHandle.getProperty(property)).jsonValue();
-  }
-
-   /**
-    * Cannot get attribute directly from a ElementHandle.
-    * If the attribute have property counterparts, use getProperty function.
-    */
-  public async getAttribute(page: Page) {
-    return await page.evaluate(link => link.getAttribute('href'), this.eHandle,);
+  public async getProp(property: string): Promise<unknown> {
+    const prop = await this.eHandle.getProperty(property);
+    console.log(`getProp(${property}) function: ${await prop.jsonValue()}`);
+    return await prop.jsonValue();
   }
 
   /**
-   * Get the element attribute value.
-   * @param {Page} page
-   * @param {ElementHandle} element
-   * @param {string} attribute
+   * Get attribute directly from a ElementHandle.
+   * If the attribute have property counterparts, use getProperty function.
    */
-  public async getAttributeAlternative(page: Page, element: ElementHandle, attribute: string) {
-    const handle = await page.evaluateHandle( (elem, attr) => {
-      return elem.getAttribute(attr);
-    }, element, attribute);
-    return await handle.jsonValue();
+  public async getAttr(attr: string): Promise<unknown> {
+    const atr = await this.eHandle.getAttribute(attr);
+    console.log(`getAttr(${attr}) function: ${atr}`);
```

Comment:
Does this still debug statement still need to be there?

Reply:
I need to see logged info for now until common functions are working consistently. eventually console.log will be all removed.